udata/core/dataset/api.py

```diff
@@ -550,3 +554,13 @@ def get(self):
         '''List all resource types'''
         return [{'id': id, 'label': label}
                 for id, label in RESOURCE_TYPES.items()]
+
+
+@ns.route('/schemas/', endpoint='schemas')
+class SchemasAPI(API):
+    @api.doc('schemas')
+    @api.marshal_list_with(schema_fields)
+    @cache.cached(timeout=60*5)
```

Comment:
I wanted to add a configuration to specify a different TTL but I didn't manage to. It seems `current_app.config` isn't available here, not in the app context.

Any way to work around this? Is it needed?

Reply:
How about `SCHEMA_CACHE_DURATION = current_app.config.get()` at the module level and then use `SCHEMA_CACHE_DURATION`? I think it should work. https://github.com/opendatateam/udata/blob/master/udata/core/dataset/preview.py#L58